packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -1522,10 +1758,12 @@ Example from a Policy object
 
 | Property     | Description                                                                                                                                                                            | Type                            | Required for create or update |
 | :----------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------ | :---------------------------- |
-| clients      | For Policies, specifies which clients are included or excluded in the Policy                                                                                                          | `include` and `exclude` lists   | True                          |
+| clients      | For Policies, specifies which clients are included or excluded in the Policy                                                                                                           | `include` and `exclude` lists   | True                          |
 | grant_type   | Can be one of the following: `authorization_code`, `password`, `refresh_token`, or `client_credentials`. Determines the mechanism Okta uses to authorize the creation of the tokens.   | Enum                            | True                          |
-| people       | For rules, specifies which Users and Groups are included or excluded in the rule                                                                                                      | `include` and `exclude` lists   | True                          |
-| scopes       | Array of Scopes this condition includes or excludes                                                                                                                                   | `include` and `exclude` lists   | True                          |
+| people       | For rules, specifies which Users and Groups are included or excluded in the rule                                                                                                       | `include` and `exclude` lists   | True                          |
+| scopes       | Array of Scopes this condition includes or excludes                                                                                                                                    | `include` and `exclude` lists   | True                          |
+
+See also the [Policy-Rule Conditions object](/docs/reference/api/policy/#conditions-object-2) section
```

Comment:
#conditions-object instead of #conditions-object-2 ?

Reply:
or maybe i'm not aware of some syntax here :|

Reply:
#conditions-object would link to the section under the policies section instead of the rules section